src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,88 @@
+package duke.task;
+
+import java.util.List;
+
+/**
+ * The TaskList class encapsulates information
+ * and methods pertaining to how Tasks are listed in Duke.
+ *
+ * @author Frederick Pek
+ * @version CS2103 AY21/22 Sem 1 iP
+ */
+public class TaskList {
+     private List<Task> tasks;
+
+     /**
+     * Creates and initalizes a new TaskList with the given list of tasks.
+     *
+     * @param tasks The list of tasks for the TaskList to be initialised with.
+     * @return A new TaskList object.
+     */
+     public TaskList(List<Task> tasks) {
+          this.tasks = tasks;
+     }
+
+     /**
+     * Retrieves and returns the Task at the specified index of the tasklist.
+     *
+     * @param index The specified index to retrieve the task from.
+     * @return The task at index.
+     */
+     public Task get(int index) {
+          return this.tasks.get(index);
```

Comment:
```suggestion
         return this.tasks.get(index);
```

Reply:
Thanks for the catch! I somehow set indentations to be 5-spaces for that particular file haha